### PR TITLE
Update docs and pass GitHub token

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
- add docs page for image generation
- help page was moved from [here](https://github.com/microsoft/azure-pipelines-image-generation/blob/master/help/CreateImageAndAzureResources.md) (I think that the file was lost after migration to virtual environments, that is why I returned it back).
- pass GitHub access token to script `helpers/GenerateResourcesAndImage.ps1`

### CODE REVIEW

File: helpers/GenerateResourcesAndImage.ps1 [hunk 1]
```diff
@@ -77,10 +80,18 @@ Function GenerateResourcesAndImage {
...
         [int] $SecondsToWaitForServicePrincipalSetup = 30,
+        [Parameter(Mandatory = $True)]
+        [string] $GithubFeedToken,
         [Parameter(Mandatory = $False)]
         [Switch] $Force
     )
 
+    if (([string]::IsNullOrEmpty($version)))
+    {
+        Write-Error "You have to specify valid GitHub PAT to dowload tool packages from GitHub Package Registry"
```
Comment (maxim-lobanov) [inline helpers/GenerateResourcesAndImage.ps1]: typo `download`

**Comment by maxim-lobanov** [comment]
@alepauly , Is there any other docs about image-generation that should be updated to notify customers about `github_feed_token` variable?

File: helpers/GenerateResourcesAndImage.ps1 (hunk 1 above)
Comment (dmitry-shibanov) [reply helpers/GenerateResourcesAndImage.ps1]: Done

**Comment by alepauly** [comment]
@maxim-lobanov, this is good. Maybe we'll need to link it from somewhere in the main readme eventually. @dmitriy-shibanov, thanks for reviving those instructions!

**Pull request merged**